scania-digital-design-system/tegel
# fix(dropdown): duplicate value in multiselect

## **Describe pull-request**  
When multiselect is set to true, choosing default values creates duplicates in placeholder. This PR enforces value uniqueness in 2 methods. Bug is visual in latest Storybook. 

## **Issue Linking:**  
- **Jira:** : [CDEP-3396](https://tegel.atlassian.net/browse/CDEP-3396)    

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to Storybook
2. Go to Dropdown component
3. Set multiselect bool to true and select multiple values from default options. Should not yield duplicates. 

## **Screenshots**  

Before:
![Skärmavbild 2024-07-24 kl  09 29 48](https://github.com/user-attachments/assets/0984c780-5469-4b04-8d0c-7577cad59a9a)

After:
![Skärmavbild 2024-07-24 kl  09 31 00](https://github.com/user-attachments/assets/1ec48feb-5bf8-483c-ade9-a840070ec3ee)



[CDEP-3396]: https://tegel.atlassian.net/browse/CDEP-3396?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
